go.mod

```diff
@@ -3,14 +3,13 @@ module github.com/minio/minio-go/v7
 go 1.12
 
 require (
-	github.com/json-iterator/go v1.1.9
+	github.com/json-iterator/go v1.1.10
 	github.com/minio/md5-simd v1.1.0
-	github.com/minio/minio-go/v6 v6.0.57 // indirect
+	github.com/minio/minio v0.0.0-20200702003552-fabd0469585a // indirect
```

Comment:
we shouldn't be depending on `minio/minio` not sure why this is coming in? @poornas